kDrive/Utils/FileActionsHelper.swift

```diff
@@ -469,7 +469,8 @@ public final class FileActionsHelper {
             floatingPanelViewController?.rightButton.isEnabled = driveFileManager.drive.accountAdmin
             floatingPanelViewController?.actionHandler = { _ in
                 driveFloatingPanelController.dismiss(animated: true) {
-                    StorePresenter.showStore(from: viewController, driveFileManager: driveFileManager)
+                    @LazyInjectService var navigationManager: NavigationManageable
```

Comment:
It is a tiny thing really, but by convention we should only use `LazyInjectService` when used as a property.